packages/custom-elements-json/schema.ts

```diff
@@ -130,6 +130,7 @@ export type Declaration =
   | ClassDeclaration
   | FunctionDeclaration
   | VariableDeclaration
+  | MixinDeclaration
```

Comment:
We should PR this here https://github.com/webcomponents/custom-elements-json/pull/9 as well